### PR TITLE
{Lab} Migrate `az lab` to AAZ

### PR DESCRIPTION
**Pull request by Jing-song**
**Related command**

`az lab`

**Description**

Migrate `az lab` to AAZ
AAZ model: https://github.com/Azure/aaz/pull/562

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Lab

**Comment by jsntcy** [comment]
> ❌AzureCLI-BreakingChangeTest
> > ❌lab
> > 
> > 
> > 
> > rule
> > cmd_name
> > rule_message
> > suggest_message
> > 
> > 
> > 
> > 
> > ❌ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > lab vm apply-artifacts
> > cmd `lab vm apply-artifacts` update parameter `name`: updated property `options` from `['--name', '-n']` to `['--name']`
> > please change property `options` from `['--name']` to `['--name', '-n']` for parameter `name` of cmd `lab vm apply-artifacts`
> > 
> > 
> > ❌ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > lab vm create
> > cmd `lab vm create` update parameter `name`: updated property `options` from `['--name', '-n']` to `['--name']`
> > please change property `options` from `['--name']` to `['--name', '-n']` for parameter `name` of cmd `lab vm create`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > lab vm list
> > cmd `lab vm list` removed parameter `order_by`
> > please add back parameter `order_by` for cmd `lab vm list`

Please check if these breaking changes are expected.
At a glance, it seems `-n` should not be removed.

**Comment by Jing-song** [comment]
> ️✔️AzureCLI-BreakingChangeTest
> > ️✔️Non Breaking Changes

Fixed

**Review by jsntcy — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/lab/validators.py [hunk 1]
```diff
@@ -533,12 +537,11 @@ def _any(collection):
 
 
 def _get_current_user_object_id(graph_client):
-    from msrestazure.azure_exceptions import CloudError
     try:
         current_user = graph_client.signed_in_user.get()
         if current_user and current_user.object_id:  # pylint:disable=no-member
             return current_user.object_id  # pylint:disable=no-member
-    except CloudError:
+    except HttpResponseError:
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/lab/validators.py:544]: `graph_client` is Track 1 SDK will never raise `HttpResponseError`.

**Pull request merged**